ext/json/ext/parser/parser.c

```diff
@@ -1488,20 +1488,42 @@ enum {JSON_integer_en_main = 1};
 #line 695 "parser.rl"
 
 
+#define MAX_FAST_INTEGER_SIZE 18
+static inline VALUE fast_parse_integer(char *p, char *pe)
```

Comment:
This isn't even a fast path to be honest, it's a very naive integer parsing function. Dunno if it's worth parsing multiple bytes at a time with `__builtin_bswap*`?

Reply:
According to a profile, in `citm_catalog` which is the benchmark with the most integers, we're spending `0.6%` in `fast_parse_integer`, and `1.8%` in the overall `JSON_parse_integer`.

So maybe not worth it.